docs/cave/d2d-gis-shapefiles.md

```diff
@@ -5,10 +5,12 @@ This section describes how to start the GIS application, configure the display a
 
```

Comment:
Overall this is a very wordy page and isn't quite written like the rest of our documentation...not sure if we should leave it or re-word it. I tried to at least add more images to go with all of the text.